docusaurus/docs/introduction/plugin-lifecycle.md

```diff
@@ -0,0 +1,93 @@
+---
+id: plugin-lifecycle
+title: Life cycle of a plugin
+description: Learn about the life cycle of a Grafana plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - lifecycle
+  - life cycle
+  - loading
+  - unloading
+  - installation
+sidebar_position: 2.5
```

Comment:
We don't have to fix this now, but suggest we multiply the sidebar positions by 10 to make simpler as in https://github.com/grafana/plugin-tools/tree/main/docusaurus/docs/tutorials

Reply:
yeah, I will make this change in a batch at a later date. for now, fractions.

Reply:
when i do that batch, that would also be a good time to fine-tune any sidebar ordering for which we want to improve the flow of topics.